src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListView.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Private.Infrastructure;
+using Uno.UI.RuntimeTests.Helpers;
+using Windows.Foundation;
+using Windows.UI.Input.Preview.Injection;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using System.Linq;
+
+namespace Uno.UI.RuntimeTests.Tests.Microsoft_UI_Xaml_Controls
+{
+	[TestClass]
+	public class Given_ListView
+	{
+		[TestMethod]
+		[RunsOnUIThread]
+#if !HAS_INPUT_INJECTOR
+		[Ignore("InputInjector is only supported on Skia #14948")]
```

Comment:
```suggestion
		[Ignore("InputInjector is not supported on this platform.")]
```

(To match the common comment that has been changed in recent version of uno as `InputInjector`is now also supported on wasm)